services/management/commands/services_import/units.py

```diff
@@ -364,6 +364,17 @@ def _import_unit(syncher, keyword_handler, info, dept_syncher,
         obj_changed = True
         obj.geometry = obj.location
 
+    if obj.municipality_id is None and obj.location is not None:
+        div = AdministrativeDivisionGeometry.objects.select_related('division', 'division__type')\
+            .filter(boundary__contains=obj.location, division__type__type='muni')
+        if div:
+            muni_name = div[0].division.name_fi.lower()
```

Comment:
Could you add a warning-level log entry when len(div) > 1?
This way we can see the warnings in our monitoring tools and maybe react if necessary.